lib/theme-options/live-options.php

```diff
@@ -0,0 +1,30 @@
+<?php
+// SITE OPTIONS
+$prefix = '_igv_';
+$suffix = '_options';
+
+$page_key = $prefix . 'live' . $suffix;
+$page_title = 'Live Page Options';
+$metabox = array(
```

Comment:
IMO is more intuitive to got to the page you want to modify, to modify it. In a case where a bunch of stuff from different pages need modification at once, yeah, site options kinda pattern works better. 

What's the case here?